src/radical/pilot/agent/rm/base.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, cfg, session):
         self.cores_per_node  = None
         self.gpus_per_node   = None
         self.lfs_per_node    = None
+        self.mem_per_node    = None
```

Comment:
We assume each node has the same amount of memory. Note that this assumption breaks when we run the agent on one of the working nodes. Depending on the total amount of memory installed, the amount of memory actually available, and the agent's memory footprint, this break of assumption might be relevant.

Reply:
I agree. But as a starting point, I think we can start with this assumption. We can relax this assumption when we have some probing mechanism in place. Would that be okay?